infrastructure/base/vars/terraform.tfvars

```diff
@@ -7,3 +7,10 @@ github_org               = "Vizzuality"
 github_repo              = "marxan-cloud"
 github_production_branch = "main"
 github_staging_branch    = "staging"
+key_vault_access_users  = ["tiago.garcia_vizzuality.com#EXT#", "andrea.rota_vizzuality.com#EXT#"]
+
+production_db_instance_size = "GP_Standard_D32d_v4"
```

Comment:
@tiagojsag just noticed this while trying to remember how many cores I could try to max out: shouldn't this be `GP_Standard_D32ds_v4`?